auction_jsligo/test/auction/bootstrap.mligo

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
need tests in jsligo, anyway, we should probably wait for validation of cameligo before spending time of transpilation